Makefile

```diff
@@ -29,6 +29,6 @@ benchmark: ## Run benchmarks
 help: ## Display this help screen
 	@grep -h -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-goversion ?= "1.12"
-test_version: ## Run tests inside Docker with given version (defaults to 1.12 oldest supported). Example: make test_version goversion=1.13
-	@docker run --rm -it -v $(shell pwd):/project golang:$(goversion) /bin/sh -c "cd /project && make check"
+goversion ?= "1.15"
```

Comment:
Guess you switched to Go 1.15 as Go 1.16 is released now. Makes sense, we should deprecate all version below Go 1.14 now anyway.

Reply:
actually 1.14 is not even supported.  

> Each major Go release is supported until there are two newer major releases. https://golang.org/doc/devel/release.html#policy